tutorials/auto_inertia_calculation.md

```diff
@@ -0,0 +1,160 @@
+density values for each and the inertia values from each would be aggregated to calculate
+the final inertia of the link. However, if there are no collisions present,
+an `ELEMENT_MISSING` error would be thrown.
+
+It is **important** to note here that if `auto` is set to `true` and the user has
+still provided values through the `<mass>`, `<pose>` and `<inertia>` tags, they
+would be **overwritten** by the automatically computed values.
+
+>**Note:** Use SDF Spec version 1.11 or greater to utilize the new tags and attributes
+of this feature.
+
+Here's an example snippet of a cylinder model that has automatic inertial calculations
+enabled and has a density of 1240 kg/m^3:
+
+```xml
+  <model name="cylinder">
+    <link name="cylinder_link">
+      <inertial auto="true" />
+      <collision name="collision">
+        <density>1240.0</density>
+        <geometry>
+          <cylinder>
+            <radius>1</radius>
+            <length>2</length>
+          </cylinder>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <cylinder>
+            <radius>1</radius>
+            <length>2</length>
+          </cylinder>
+        </geometry>
+        <material>
+          <diffuse>1.0 1.0 0.0 1.0</diffuse>
+          <ambient>1.0 1.0 0.0 1.0</ambient>
+          <specular>1.0 1.0 0.0 1.0</specular>
+        </material>
+      </visual>
+    </link>
+  </model>
+```
+
+If you use the above snippet in an empty world and launch it with `gz-sim`, here's
+how it would look:
+
+![Cylinder](files/auto_inertia/cylinder_inertia_demo.gif)
+
+## Links with Multiple Collisions & the Effect of Density
+
+To understand the inertia calculation in links with multiple collisions and the
+effect of setting different density values, you can launch the `auto_inertia_pendulum.sdf`
+example world using:
+
+```bash
+gz sim auto_inertia_pendulum.sdf
+```
+
+After the gz-sim window opens up, you can right click on both the models and enable
+the centre of mass visualization by selecting the `View > Center of Mass` option from
+the menu. Once you play the simulation it should look this:
+
+![Pendulum](files/auto_inertia/auto_inertia_pendulum.gif)
+
+This example world has two structurally indentical models. The pendulum link of both
+the models contain 3 cylindrical collision geometries: One on the top which forms the
+joint, One in longer cylinder in middle and One at the end which forms the bob of
+cylinder. Even, though they are indentical, the center of mass for both are different
+as they use different density values for the different cylinder collisions. One one
+hand, the upper joint collision of the pendulum on the left has the highest density
+which causes the center of mass to shift closer to the axis while on the other hand,
+the bob collision of the pendulum on the right has the highest density which causes
+the center of mass to shift towards the end of the pendulum.

```

Comment:
```suggestion
as they use different density values for the different cylinder collisions. On one
hand, the upper joint collision of the pendulum on the left has the highest density
which causes the center of mass to shift closer to the axis. While on the other hand,
the bob collision of the pendulum on the right has the highest density which causes
the center of mass to shift towards the end of the pendulum.

```

Reply:
Corrected in [2b8be4c](https://github.com/gazebosim/gz-sim/pull/2119/commits/2b8be4cfc423ee3b6ec37763861572a4362ec10b)